corehq/apps/domain/forms.py

```diff
@@ -135,6 +138,8 @@
 # used to resize uploaded custom logos, aspect ratio is preserved
 LOGO_SIZE = (211, 32)
 
+upload_size_limit = f"{settings.MAX_UPLOAD_SIZE_ATTACHMENT/(1024*1024):,.0f}"
```

Comment:
what syntax is that? Nevermind I asked copilot and I got:

:,.0f: This is a format specification for the expression. The : character starts the format specification. The , option signals the use of a comma as a thousand separator. The .0f specifies floating point format with 0 digits after the decimal point. So, if the result of the division was 1234.5678, it would be formatted as 1,235.

🤯